lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -2224,10 +2224,6 @@ namespace Js
             &parserStateCacheSize, dwFlags);
         END_TEMP_ALLOCATOR(tempAllocator, this);
 
```

Comment:
Do we have a potential leak here, since we're still calling CoTaskMemAlloc below?

Reply:
I changed the ByteCodeBufferBuilder to allocate the buffer out of scriptContext->SourceCodeAllocator() when we're generating a parser state cache because the list of serialized ids of captured names are indices in the bytecode buffer's string table. So in that case we want the buffer to be kept around for when those functions are undeferred. Does that make sense?

Reply:
Close enough. :)